MaxText/inference/page_manager.py

```diff
@@ -22,6 +22,8 @@
 similar to virtual memory systems.
 """
 
+# TODO(rupliu) Add unit tests for this file
```

Comment:
Just adding a note that these tests largely exist already, but may need some tweaking when added.

https://github.com/AI-Hypercomputer/maxtext/blob/page_manager/MaxText/tests/page_manager_test.py.

Reply:
thanks, updated!